examples/pyrightconfig.json

```diff
@@ -0,0 +1,4 @@
+{
+    "typeCheckingMode": "strict",
+    "exclude": ["defer_import_slow_startup/defer_import/mean.py"],  // because of the pandas import
```

Comment:
Are you sure you don't want to change `defer_import_slow_startup/defer_import/mean.py`
to use `statistics.mean` instead of `pandas`?
Then we wouldn't have to deal with this missing import problem.

Reply:
i think the problem is moreso that executing the examples from the venv of the main project simply doesnt work. it complains about relative paths and such. something like `uv run --directory examples/defer_import_slow_startup -- basedpyright -p ../pyrightconfig.json defer_import` fixes these issues